docker/Makefile

```diff
@@ -0,0 +1,26 @@
+## Build container
+build:
+	@bash -x scripts/build.sh
```

Comment:
If `build.sh` is a one-liner, just inline it here.

Reply:
Yeah, I was kind of wondering if it is more common for Docker users to invoke `make -C docker build`, `./docker/scripts/build.sh`, or something else.

For `make -C docker run` I would assume you'll want a way to pass arguments, so that the user can at least perform basic operations like importing a new token?

Reply:
Honestly I think it's more common for Docker users to run docker commands themselves. If I saw a Dockerfile in a project, it would be logical that I can simply do

    docker build -t mytagname dir
    docker run -it mytagname bash

I'm not sure these make targets or helper scripts provide much value.

Reply:
In short, as a first step, I would probably just commit docker/Dockerfile and leave it at that. Most Docker users and tools know what to do with that.